rocAL/rocAL_pybind/amd/rocal/plugin/pytorch.py

```diff
@@ -215,7 +275,7 @@ def __len__(self):
         return self.len
 
     def __del__(self):
-        b.rocalRelease(self.loader._handle)
+        # b.rocalRelease(self.loader._handle)
```

Comment:
Why rocAL release has been commented out?

Reply:
The fix for hang was in another repo. Added that fix here & tested

Reply:
Uncommented this out